assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -75,6 +75,7 @@ function getRootMargin( breakpoint ) {
 	return `${ -top }px ${ -gap }px ${ -gap }px ${ -gap }px`;
 }
 
+/* eslint-disable complexity */
```

Comment:
This is file is big, and new edit I made for 1 more ternary operator made complexity reach 21. Since it is an old file and pretty big, can't refactor it as part of this PR which is already pretty big (and that would include more testing around general functionality of widget areas, etc). And this small change does not affect actual readability of the file

Reply:
@zutigrm, could you please create a new issue for this?

Reply:
Actually, after thinking a bit more, I am not sure that widget area changes should be in this PR. In this ticket we need to update the setup banner only, anything else is out of scope. Could you please revert these changes and create a new ticket to update the subtitle?

Reply:
That's a valid point. Was considering this initially, but since it wasn't big change thought to include it along here since it might be considered as part of the change. But you are correct, it makes more sense to do this separately. I created #9444 as part of the epic and follow up on this issue

Thanks

Reply:
Thanks a lot.